swagger.yaml

```
@@ -264,6 +264,412 @@ paths:
+      description: "Patch a field in a discussion by id"
+      produces:
+        - "application/json"
+      parameters:
+        - in: header
+          name: Columbus-User-Email
+          type: string
+          format: email
+          required: true
+        - in: path
+          name: id
+          description: "discussion's ID"
+          type: integer
+          required: true
+      responses:
+        204:
+          description: No Content
+        400:
+          description: validation error
+          schema:
+            $ref: "#/definitions/ValidationError"
+        404:
+          description: "not found error"
+          schema:
+            $ref: "#/definitions/Error"
+        500:
+          description: "internal server error"
+          schema:
+            $ref: "#/definitions/InternalServerError"
+  "/v1beta1/discussions/{discussion_id}/comments":
+    get:
+      tags:
+        - Comments
+      summary: Get list of comments of a discussion
+      description: "Returns list of comments of a discussion"
+      produces:
+        - "application/json"
+      parameters:
+        - in: header
+          name: Columbus-User-Email
+          type: string
+          format: email
+          required: true
+        - in: path
+          name: discussion_id
+          description: "discussion's ID"
+          type: integer
+          required: true
+        - in: query
+          name: sort
+          schema:
+            type: string
+            enum:
+              - created_at
+              - updated_at
+            default: created_at
+        - in: query
+          name: direction
+          schema:
+            type: string
+            enum:
+              - asc
+              - desc
+            default: desc
+        - in: query
+          name: size
+          type: integer
+        - in: query
+          name: offset
+          type: integer
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: "#/definitions/Comment"
+        400:
+          description: validation error
+          schema:
+            $ref: "#/definitions/ValidationError"
+        500:
+          description: "internal server error"
+          schema:
+            $ref: "#/definitions/InternalServerError"
+    post:
+      tags:
+        - Comments
+      summary: Create a comment of a discussion
+      description: "Create a comment of a discussion"
+      produces:
+        - "application/json"
+      parameters:
+        - in: header
+          name: Columbus-User-Email
+          type: string
+          format: email
+          required: true
+        - in: path
+          name: discussion_id
+          description: "discussion's ID"
+          type: integer
+          required: true
+        - in: body
+          name: ""
+          required: true
+          schema:
+            allOf:
+              - $ref: "#/definitions/Comment"
+              - required:
+                  - body
```

Since only `body` field is required here let's just change it like this to prevent misleading user having to prepare all other fields.

```suggestion
            type: object
            properties:
              body:
                type: string
                example: "This is my first comment"
                required: true
```